frontend/src/components/logged_in/ConfirmableButton.js

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+
+function ConfirmableButton({ text, onConfirm, className, style }) {
+  const [confirmDelete, setConfirmDelete] = useState(false);
+
+  function handleMouseLeave() {
+    setConfirmDelete(false);
+  }
+
+  function handleClick() {
```

Comment:
i like these buttons. this is a cool idea. Time permitting we should probably do this for all the delete buttons. Cool stuff!